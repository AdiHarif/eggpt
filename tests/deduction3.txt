Cats are afraid of wolves.
Winona is a cat.
Emily is a cat.
Mice are afraid of wolves.
Wolves are afraid of mice.
Sheep are afraid of cats.
Gertrude is a mouse.
Jessica is a mouse.
What is gertrude afraid of?
What is emily afraid of?	
What is gertrude afraid of?
What is emily afraid of?	