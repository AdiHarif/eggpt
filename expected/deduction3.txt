What is gertrude afraid of?	wolf	7 4
What is emily afraid of?	wolf	3 1
What is gertrude afraid of?	wolf	7 4
What is emily afraid of?	wolf	3 1